templates/home.html

```diff
@@ -4,14 +4,16 @@ <h1> Welcome </h1>
 {% endblock %}
 
 {% block body %}
+{% block nav %}
 <table>
     <td>Home</td>
     <td>Search</td>
```

Comment:
add a route and link to the search template which should extend home.html and have a form to search